app/controllers/spree/user_registrations_controller.rb

```diff
@@ -88,4 +95,14 @@ def spree_user_params
   def after_sign_in_redirect(resource_or_scope)
     stored_location_for(resource_or_scope) || account_path
   end
+
+  def redirect_to_checkout_or_account_path(resource)
+    resource_path = after_sign_up_path_for(resource)
```

Comment:
?
```suggestion
    resource_path = after_inactive_sign_up_path_for(resource)
```

Reply:
If I change this method than after sign up in `checkout/registration` I will be redirected to account_path and not back to `checkout/address path`